### ISSUE TITLE
I encountered the bug after two rotations.

### ISSUE THREAD

**Issue by poyotamu000** [euslisp/EusLisp#499]
error message
```
irteusgl 0 error: undefined function geometry::rotation in (send b :rotate-vertices pi/2 #f(0.0 0.0 1.0))
```

code
```
1.irteusgl$ (setq b (make-cube 10 10 10))
#<body #X562aca24e1b0 (:cube 10.0 10.0 10.0) 0.0 0.0 0.0 / 0.0 0.0 0.0>
2.irteusgl$ (send b :rotate-vertices pi/2 :z)
#<body #X562aca24e1b0 (:cube 10.0 10.0 10.0) 0.0 0.0 0.0 / 0.0 0.0 0.0>
3.irteusgl$ (send b :rotate-vertices pi/2 #f(0 0 1))
Call Stack (max depth: 20):
  0: at (send b :rotate-vertices pi/2 #f(0.0 0.0 1.0))
  1: at (send b :rotate-vertices pi/2 #f(0.0 0.0 1.0))
  2: at #<compiled-code #X562ac9e1a3e8>
```

![Screenshot from 2023-03-28 18-09-52](https://user-images.githubusercontent.com/49392029/228188290-e6c1bdf0-9307-4f2f-bc19-fbaf30fec12f.png)

**Comment by tkmtnt7000** [comment]
cc: @tkmtnt7000

**Comment by tkmtnt7000** [comment]
Maybe correct.
```lisp
16.irteusgl$ load "geobody.l"
t
17.irteusgl$ (setq b (make-cube 10 10 10))
#<body #X563f4d6e2a70 (:cube 10.0 10.0 10.0) 0.0 0.0 0.0 / 0.0 0.0 0.0>
18.irteusgl$ (send b :rotate-vertices pi/2 :z)
#<body #X563f4d6e2a70 (:cube 10.0 10.0 10.0) 0.0 0.0 0.0 / 0.0 0.0 0.0>
19.irteusgl$ (send b :rotate-vertices pi/2 #f(0 0 1))
#<body #X563f4d6e2a70 (:cube 10.0 10.0 10.0) 0.0 0.0 0.0 / 0.0 0.0 0.0>

```

**Comment by k-okada** [comment]
closes via #500 #504
